google-cloud-spanner/src/test/java/com/google/cloud/spanner/StatementTest.java

```diff
@@ -103,33 +101,47 @@ public void bindReplacement() {
   public void incompleteBinding() {
     Statement.Builder builder = Statement.newBuilder("SELECT @v");
     builder.bind("v");
-    expectedException.expect(IllegalStateException.class);
-    builder.build();
+    try {
+      builder.build();
+      fail("");
```

Comment:
nit: Here and below, please add a text like "Expected exception"

Reply:
Done